lib/ansible/modules/network/enos/enos_config.py

```diff
@@ -248,32 +243,35 @@ def main():
         after=dict(type='list'),
 
         match=dict(default='line', choices=['line', 'strict', 'exact', 'none']),
-        replace=dict(default='line', choices=['line', 'block', 'config']),
+        replace=dict(default='line', choices=['line', 'block']),
 
         config=dict(),
+        defaults=dict(type='bool', default=False),
+        passwords=dict(type='bool', default=False),
+
         backup=dict(type='bool', default=False),
-        comment=dict(default=DEFAULT_COMMIT_COMMENT),
-        admin=dict(type='bool', default=False)
+        save=dict(type='bool', default=False),
```

Comment:
`DOCUMENTATION:` for `save:` needs adding.

Reply:
Reverting the changes back.